labs/advanced-topics/stop.sh

```diff
@@ -1,8 +1,12 @@
-#! /bin/bash
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
 
```

Comment:
Reordering the stop commands like this means that the script will still work when there are no consumers running.